lib/ZuluSCSI_platform_RP2040/ZuluSCSI_platform_msc.cpp

```diff
@@ -0,0 +1,236 @@
+/**
+ * Copyright (c) 2023-2024 zigzagjoe
+ * 
+ * ZuluSCSI™ firmware is licensed under the GPL version 3 or any later version. 
+ * 
+ * https://www.gnu.org/licenses/gpl-3.0.html
+ * ----
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version. 
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details. 
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+**/
+
+/* platform specific MSC routines */
+#ifdef PLATFORM_MASS_STORAGE
+
+#include <SdFat.h>
+#include <device/usbd.h>
+#include <hardware/gpio.h>
+#include "ZuluSCSI_platform.h"
+#include "ZuluSCSI_log.h"
+#include "ZuluSCSI_msc.h"
+
+#include "msc.h"
+#include "msc_device.h"
```

Comment:
Unfortunately not - this is required as we must recompile msc_device.c with a greater value for CFG_TUD_MSC_EP_BUFSIZE as the framework defines it to 64 bytes which /will not/ work. 

If you have a cleaner way to sort that out, I'm all ears...

Reply:
So is `tusb_config.h` ignored?
I think if we add the define `-DPIO_FRAMEWORK_ARDUINO_NO_USB`
and switch the USB serial logging to the TinyUSB implementation instead of the Arduino version it might work

Reply:
The framework has its own tusb_config.h which is used https://github.com/earlephilhower/arduino-pico/blob/master/include/tusb_config.h and unilaterally sets it to 64 bytes. 

It's really a flaw in the framework, IMO, it would be preferential to have it do an #ifndef  CFG_TUD_MSC_EP_BUFSIZE so it could be overridden on the commandline or something.

Reply:
Since all RP2040/PICO builds will be using MSC and we have a custom spin of the arduinopico package, I'm fine with changing the `tusb_config.h` file with the bigger CFG_TUD_MSC_EP_BUFSIZE in the arduinopico package. That way we don't have to have to maintain any TinyUSB code.

Reply:
@ZigZagJoe 
 I changed the `tusb_config.h` in the framework-arduinopico package and rebuilt the static libraries. 
You'll probably have to delete `~/.platformio/packages/framework-arduinopico` to get the new static libraries.
You should be able now delete `msc.h`, `msc_device.c`, and `msc_device.h` and change their include paths to 
```
#include "class/msc/msc.h"
#include "class/msc/msc_device.h"
```

Reply:
You can also delete the `tusb_config.h` from `lib/ZuluSCSI_platform_RP2040/` and add
```
#include <tusb_config.h>
#include <class/msc/msc.h>   // I figure we changed these to brackets as we are referencing external code
#include <class/msc/msc_device.h>
```

Reply:
yep, i'm just waiting on a validation build after the framework updates

Reply:
I did verify that the new spin of [v3.6.0-DaynaPORT](https://github.com/rabbitholecomputing/arduino-pico/tree/v3.6.0-DaynaPORT]
branch works the `ZuluSCSI_Pico_DaynaPORT_MSC`